syncthing_gtk/editordialog.py

```diff
@@ -54,7 +54,8 @@ def __init__(self, app, uifile, title):
         self.setup_widgets(uifile, title)
         # Move entire dialog content to ScrolledWindow if screen height
         # is too small
-        if Gdk.Screen.get_default().height() < 900:
+        #if Gdk.Display.get_default().height() < 900:
+        if False:
```

Comment:
Looks like a temp hack. It should either be resolved before merging, or at least marked with a TODO comment.